KalturaWowzaServer/build.gradle

```diff
@@ -63,6 +63,7 @@ dependencies {
             "$WMSINSTALL_HOME/lib/joda-time-2.3.jar",
             "$WMSINSTALL_HOME/lib/commons-lang-2.6.jar",
             "$WMSINSTALL_HOME/lib/jid3lib-0.5.4.jar",
+            "$WMSINSTALL_HOME/lib/commons-lang3-3.4.jar"
```

Comment:
Pair is only supported since Lang 3. We need to choose between replacing commons-lang-2.6.jar by commons-lang3-3.4.jar that comes with wowza 4.6 or replace Pair by other class.